### PR TITLE
(DOCS-2753) Remove instances of e.g. from the docs

### PR DESCRIPTION
**Pull request by jtappa**
### What does this PR do?
Replaces e.g. and its variants (and a couple other linter items) in the docs.

### Motivation
OKR

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security/_index.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ further_reading:
...
+Datadog allows customers to submit data in multiple ways, including from the [Agent][1], the [public API][2], and [integrations][3]. This article describes the main categories of data which might be submitted by customers to Datadog as part of the intended use of its product, and highlights scenarios where submitted data may contain personal data. Read the [security page][4] and [privacy policy][5] for more information.
 
 ## Metadata
 
-Metadata consist primarily of [tags][6], which are typically formatted in the `key:value` (e.g. `env:prod`) format. Metadata enable customer data such as Infrastructure Metrics, APM and Logs to be filtered and grouped. Metadata should not contain personal data as part of the intended use of the service.
+Metadata consist primarily of [tags][6], which are typically formatted in the `key:value` (for example, `env:prod`) format. Metadata enable customer data such as Infrastructure Metrics, APM and Logs to be filtered and grouped. Metadata should not contain personal data as part of the intended use of the service.
```
Comment (alai97) [inline content/en/security/_index.md:28]:
```suggestion
Metadata consist primarily of [tags][6], which are typically formatted in the `key:value` (for example, `env:prod`) format. Metadata enable customer data such as Infrastructure Metrics, APM, and Logs to be filtered and grouped. Metadata should not contain personal data as part of the intended use of the service.
```

File: content/en/security/agent.md [hunk 2]
```diff
@@ -40,15 +40,15 @@ The Datadog Agent submits data to Datadog over a TLS-encrypted TCP connection by
...
 
 ## Agent logs obfuscation
 
-The Datadog Agent generates local logs in order to support [Agent troubleshooting][11] as required. As a safety precaution, these local logs are filtered for some specific keywords and patterns that could indicate a potential credential (e.g. API key, password and token keywords, etc.), which are then obfuscated before being written to disk.
+The Datadog Agent generates local logs in order to support [Agent troubleshooting][11] as required. As a safety precaution, these local logs are filtered for some specific keywords and patterns that could indicate a potential credential (for example, API key, password and token keywords, etc.), which are then obfuscated before being written to disk.
```
Comment (alai97) [inline content/en/security/agent.md:47]:
```suggestion
The Datadog Agent generates local logs in order to support [Agent troubleshooting][11] as required. As a safety precaution, these local logs are filtered for some specific keywords and patterns that could indicate a potential credential (for example, API key, password, and token keywords), which are then obfuscated before being written to disk.
```

File: content/en/security_platform/security_monitoring/log_detection_rules.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ Joining together logs that span a timeframe can increase the confidence or sever
 
 {{< img src="security_platform/security_monitoring/detection_rules/joining_queries_define.png" alt="Define search queries"  >}}
 
-The Detection Rules join the logs together using a group by value. The group by values are typically entities (e.g. IP address, user, etc), but can be any attribute.
+The Detection Rules join the logs together using a group by value. The group by values are typically entities (for example IP address, or user), but can be any attribute.
```
Comment (alai97) [inline content/en/security_platform/security_monitoring/log_detection_rules.md:78]:
```suggestion
The Detection Rules join the logs together using a group by value. The group by values are typically entities (for example, IP address or user), but can be any attribute.
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Three small copy nits!

**Comment by jtappa** [comment]
All suggestions applied! TY!!

**Review by alai97 — APPROVED** [review]

**Pull request merged**